acl-lib/test/cmr/acl/test/acl_fetcher_test.clj

```diff
@@ -0,0 +1,72 @@
+(ns cmr.acl.test.acl-fetcher-test
+  "test functions in acl-fetcher"
+  (:require
+    [clojure.test :refer [deftest is testing]]
+    [cmr.acl.acl-fetcher :as acl-fetcher]))
+
+(def object-identity-types-ex
+  [:system-object :provider-object :single-instance-object])
+
+(deftest acl-cache-key-test
+  (is (= acl-fetcher/acl-cache-key :acls)))
+
+(deftest acl-keys-to-track-test
+  (is (= acl-fetcher/acl-keys-to-track [":acls-hash-code"])))
+
+(deftest create-acl-cache
+  (let [acl-cache (acl-fetcher/create-acl-cache [:system-object :provider-object :single-instance-object])]
+    (is (instance? cmr.common.cache.single_thread_lookup_cache.SingleThreadLookupCache acl-cache))))
+
+(deftest identity-string-map-test
+  (let [expected-map {:system-object "system"
+                      :provider-object "provider"
+                      :single-instance-object "single_instance"
+                      :catalog-item "catalog_item"}]
+    (is (= acl-fetcher/identity-string-map expected-map))))
+
+(deftest object-identity-types->identity-strings-test
+  (let [fun #'cmr.acl.acl-fetcher/object-identity-types->identity-strings]
+    (is (= (fun object-identity-types-ex) ["system" "provider" "single_instance"]))))
+
+(deftest context->cached-object-identity-types-test
+  (let [acl-cache (acl-fetcher/create-consistent-acl-cache object-identity-types-ex)
```

Comment:
Can we create the acl-cache as a :once use-fixture since it's used in multiple tests with the same args?

Reply:
Actually look back at this test, it doesn't even need the cache in the context. And the contexts are different for every test, so may not need to use a fixture